crates/extensions/c8y_http_proxy/src/actor.rs

```diff
@@ -78,6 +79,15 @@ impl Actor for C8YHttpProxyActor {
 
         while let Some((client_id, request)) = self.peers.clients.recv().await {
             let result = match request {
+                C8YRestRequest::GetJwtToken(_) => {
+                    self.get_jwt_token().await.map(|response| response.into())
+                }
+
+                C8YRestRequest::IsUrlInCurrentDomain(request) => self
+                    .url_is_in_my_tenant_domain(request)
+                    .await
+                    .map(|response| response.into()),
```

Comment:
This can be removed as the mapper is using it's own c8y-end-point to make this check.